src/commands/branches.ts

```diff
@@ -189,17 +189,49 @@ export const builder = (argv: yargs.Argv) =>
       async (args) => await get(args as any),
     )
     .command({
-      command: 'schema-diff [base-branch] <compare-branch>',
+      command: 'schema-diff [base-branch] [compare-source[@(timestamp|lsn)]]',
       aliases: ['sd'],
-      describe: 'Compare schemas from two branches',
-      builder: (yargs) =>
-        yargs.options({
-          database: {
-            type: 'string',
-            description:
-              'Name of the database for which the schema is retrieved',
-          },
-        }),
+      describe:
+        'compare a branch schema to a specific point in time' +
```

Comment:
```suggestion
        'Compare the latest schemas of any two branches, or compare a branch to its own or another branch's history.' +
```

Reply:
This suggested copy is longer, but I think gets at the value of schema diff a bit more clearly.